### PR TITLE
IEP-455: README_CN.md updated with required images and screenshots

### PR DESCRIPTION
**Pull request by alirana01**
README_CN.md updated with required images and screenshots

### CODE REVIEW

**Review by kolipakakondal — APPROVED** [review]
LGTM. 

We can wait for Natasha to review the screenshots.

**Comment by NatashaLi-ESPRESSIF** [comment]
Hi @kolipakakondal and @alirana01 , 

I have the following suggestions/questions for discussions:

1. Add instructions on how to switch between CN/EN plugin interface in the readme file, once iep 449 is merged;

2. There are mixed English and Chinese text in some screenshots, for example, in the "create a new project", "create a new project using ESP-IDF templates", "viewing serial output", "application size analysis - overview" section. Also, texts such as "Next" and "Finish" are not translated into Chinese anywhere. 

3. Not all the screenshots for plugin are replaced with Chinese languages. I understand probably we only replace screenshot within the scope of plugin interface, not eclipse. There are still a couple of them I am not sure about. For example, in the "compiling the project", "cmake editor", "ESP-IDF terminal", "sdk configuration editor", "came editor", "application size analysis - details",  and "error log" sections. 

Thank you for putting everything together! Have a great day : )

**Comment by alirana01** [comment]
> Hi @kolipakakondal and @alirana01 ,
> 
> I have the following suggestions/questions for discussions:
> 
> 1. Add instructions on how to switch between CN/EN plugin interface in the readme file, once iep 449 is merged;
> 2. There are mixed English and Chinese text in some screenshots, for example, in the "create a new project", "create a new project using ESP-IDF templates", "viewing serial output", "application size analysis - overview" section. Also, texts such as "Next" and "Finish" are not translated into Chinese anywhere.
> 3. Not all the screenshots for plugin are replaced with Chinese languages. I understand probably we only replace screenshot within the scope of plugin interface, not eclipse. There are still a couple of them I am not sure about. For example, in the "compiling the project", "cmake editor", "ESP-IDF terminal", "sdk configuration editor", "came editor", "application size analysis - details",  and "error log" sections.
> 
> Thank you for putting everything together! Have a great day : )

Hi @NatashaLi-ESPRESSIF 

Can you please reverify the changes also the changing language is added to documentation in the following PR [(https://github.com/espressif/idf-eclipse-plugin/pull/297)]

We need to translate that as well so if you can please provide translation for it against the Jira ticket created we can add that to IEP-449 PR and go for the merge on that and then this.

Thanks

**Pull request merged**
